### PR TITLE
New - AAT Schemas (fixes: #1)

### LINKED ISSUE #1: Create v1 AAT schema

### PR DESCRIPTION
**Pull request by joe-replin**
### What has changed?

- Fixes #1 
- Slight EL Lint Adjustments
- New - v1 AAT Schemas

### Blockers
- Still requires a thorough test in an AAT environment.

### CODE REVIEW

**Comment by joe-replin** [comment]
@danielghost, should this be compatible with legacy AAT? Also, does _connectionTest need to be enable for this to function as intended?

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

**Comment by joe-replin** [comment]
This PR is also in relation to the implementation of the following PR: https://github.com/adaptlearning/adapt-contrib-spoor/pull/270

File: js/errorNotificationManager.js [hunk 1]
```diff
@@ -64,14 +64,14 @@ export default class ErrorNotificationManager extends Backbone.Model {
 
     let isCancellable = true;
 
-    if (config.hasOwnProperty('_isCancellable')) {
+    if (Object.prototype.hasOwnProperty.call(config, '_isCancellable')) {
```
Comment (danielghost) [inline js/errorNotificationManager.js]:
```suggestion
    if (Object.hasOwn(config, '_isCancellable')) {
```
`hasOwnProperty` can be called directly on the object so doesn't need this syntax, although we should probably switch to `Object.hasOwn` instead.

Comment (oliverfoster) [reply js/errorNotificationManager.js]: Unforunately `hasOwn` doesn't seem to be supported by our current version of eslint, but it does work with transpilation, so it's fine to use, even if it seems unavailable in the intellisense dropdowns.

**Review by oliverfoster — APPROVED** [review]

**Pull request merged**